src/main/java/com/carhouse/provider/impl/CarFeatureProviderImpl.java

```diff
@@ -13,19 +16,22 @@
 @Component
 public class CarFeatureProviderImpl implements CarFeatureProvider {
 
+    @Value("${carSale.url}")
+    private String URL;
+
+    @Value("${car.feature.list.get}")
```

Comment:
base endpoint url has name `carSale.url`.
How can I understand that this property is a part of path for that endpoint?
Read your properties names carefully like you read it first time in your life and try to guess what this application created for.

Reply:
I'll be using properties: `carSale.url.host` and `carSale.url.port`